xysched/README.md

```diff
@@ -79,7 +91,8 @@ future.Catch(func(e error) { fmt.Println(e) })
 xysched.Now() <- future
 ```
 
-6. Create a new scheduler if it is necessary.
+6.  Create a new scheduler if it is necessary.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
6. Create a new scheduler if it is necessary.
```